Makefile

```diff
@@ -76,7 +76,7 @@ test:
 test-v:
 	go test -race -v -short -tags='debug testing' -timeout=15s $(pkgs) -run=$(run)
 test-long: clean fmt vet lint
-	go test -v -race -tags='testing debug' -timeout=500s $(pkgs) -run=$(run)
+	go test -v -race -tags='testing debug' -timeout=800s $(pkgs) -run=$(run)
```

Comment:
I think Travis auto-fails if there's no output after a certain amount of time (10 mins?) but our tests should print frequently enough to prevent it